src/mscorlib/src/System/Globalization/SortKey.cs

```diff
@@ -55,25 +55,6 @@ internal SortKey(String localeName, String str, CompareOptions options, byte[] k
         }
 
 #if FEATURE_USE_LCID
-        [OnSerializing]
-        private void OnSerializing(StreamingContext context)
-        {
-            //set LCID to proper value for Whidbey serialization (no other use)
-            if (win32LCID == 0)
-            {
-                win32LCID = CultureInfo.GetCultureInfo(localeName).LCID;
-            }
-        }
-
-        [OnDeserialized]
-        private void OnDeserialized(StreamingContext context)
-        {
-            //set locale name to proper value after Whidbey deserialization
-            if (String.IsNullOrEmpty(localeName) && win32LCID != 0)
-            {
-                localeName = CultureInfo.GetCultureInfo(win32LCID).Name;
-            }
-        }
```

Comment:
>  Why do we have two near identical copies of SortKey?

We have two copies of globalization:
- For Windows, we use the old one from desktop
- For Unix, we use the new one from CoreRT.

Ideally, we would use the one from CoreRT on Windows as well, but it needs some work... .